nb/ide.launcher/windows/nblauncher.h

```diff
@@ -64,7 +64,6 @@ class NbLauncher {
     virtual bool initBaseNames();
     virtual void addSpecificOptions(CmdArgs &args);
     virtual bool areWeOn32bits();
```

Comment:
`areWeOn32bits` seems to exist just for `adjustHeapAndPermGenSize` so you should delete that one too.